openscenario/openscenario_interpreter/include/openscenario_interpreter/syntax/unsigned_short.hpp

```diff
@@ -16,16 +16,17 @@
 #define OPENSCENARIO_INTERPRETER__SYNTAX__UNSIGNED_SHORT_HPP_
 
 #include <iostream>
-#include <std_msgs/msg/u_int16.hpp>
 #include <string>
 
 namespace openscenario_interpreter
 {
 inline namespace syntax
 {
-struct UnsignedShort : public std_msgs::msg::UInt16
+struct UnsignedShort
 {
-  using value_type = decltype(std_msgs::msg::UInt16::data);
+  using value_type = std::uint16_t;
+
+  value_type data = 0U;
```

Comment:
From N4659 11.6 (6.1)

> To zero-initialize an object or reference of type T means:
> if T is a scalar type (6.9), the object is initialized to the value obtained by converting the integer literal
0 (zero) to T

therefore the following code is sufficient:

```suggestion
  value_type data;
```